test/parallel/test-util-inspect.js

```diff
@@ -9,6 +9,7 @@ assert.strictEqual(util.inspect(false), 'false');
 assert.strictEqual(util.inspect(''), "''");
 assert.strictEqual(util.inspect('hello'), "'hello'");
 assert.strictEqual(util.inspect(function() {}), '[Function]');
+assert.strictEqual(util.inspect(async function() {}), '[async Function]');
```

Comment:
The original uses the capitalized constructor name (`Function`). How about using the same format, `AsyncFunction`?